packages/grafana-ui/src/components/Tooltip/Tooltip.story.tsx

```diff
@@ -13,16 +13,57 @@ export default {
     docs: {
       page: mdx,
     },
+    knobs: {
+      disabled: true,
+    },
+  },
+  argTypes: {
+    content: {
+      control: {
+        type: 'text',
+      },
+    },
+    theme: {
+      control: {
+        type: 'select',
+        options: ['info', 'error'],
+      },
+    },
+    placement: {
+      control: {
+        type: 'select',
+        options: [
+          'auto',
+          'bottom',
+          'top',
+          'auto-start',
+          'auto-end',
+          'right',
+          'left',
+          'top-start',
+          'top-end',
+          'bottom-start',
+          'bottom-end',
+          'right-start',
+          'right-end',
+          'left-start',
+          'left-end',
+        ],
+      },
+    },
   },
 };
 
-export const basic = () => {
-  const VISUAL_GROUP = 'Visual options';
-  // ---
-  const theme = select('Theme', ['info', 'error', 'info-alt'], 'info', VISUAL_GROUP);
+export const Basic: Story = ({ content, ...args }) => {
```

Comment:
```suggestion
export const Basic: Story = ({ args }) => {
```
There's no need to extract `content` here, as you can just pass all the args to the Tooltip.